src/components/NavBar.js

```diff
@@ -99,7 +99,7 @@ class NavBar extends Component {
               : (
                 <Nav className="ml-auto">
                   <Button
-                    style={{ fontFamily: 'AppleSDGothicNeo-Bold', color: '#333333', height: '35px' }}
+                    style={{ fontFamily: 'AppleSDGothicNeo-Bold', color: '#333333', height: 'auto' }}
```

Comment:
At some point we should change the font family, I don't think this "apple" font works on windows

Reply:
I'll take care of it. But usually if a particular font isn't available, the browser will just select a default one.

Reply:
The best practices is to keep the fonts consistent to all browsers and devices. We should use some common fonts. But we can care about this later